.eslintrc.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
     browser: true,
   },
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: 'latest',
```

Comment:
I'm not sure that we need to pin `2020` anymore — I only ended up momentarily scratching my head when bumping the eslint version failed to fix my linting issues. If you think it'd be better, we can just re-pin to `2022` rather than `'latest'` 🤷

Reply:
I suppose I would prefer to use an explicit version so that it's pinned at some moment in time

Reply:
Sure, I changed it to `2022`.